application/forms/TimeRangePicker/CommonForm.php

```diff
@@ -126,16 +127,13 @@ protected function createSelect($name, $label, $description, array $options, $si
             'label'         => $label,
             'description'   => $description,
             'multiOptions'  => $multiOptions,
+            'title' => $description,
```

Comment:
Please indent the arrow like the ones above altering the origin commit.

Reply:
Rebased and should be solved now.